packages/itwinui-variables/index.scss

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './mixins';
+
+:root {
+  @include light-variables;
```

Comment:
would it make sense to add `theme` ?
`light-theme-variables`

Reply:
it's just a name of an internal mixin and it's one of the two files in the repo, so it doesn't really matter. but i can change it if you want